framework/doc/content/source/interfaces/Coupleable.md

```diff
@@ -1,31 +1,57 @@
-Similarly,
+This class provides an API for coupling different kinds of variables values into MOOSE systems.
+The following tables summarize the methods it provides.
+
+| Methods for scalar variables | Description |
+| :--- | :--- |
+`coupledValue`^*‡§^ | Value of a coupled variable at q-points
+`coupledGradient`^*‡§^ | Gradient of a coupled variable at q-points
+`coupledSecond`^*‡§^ | Second derivatives of a coupled variable at q-points
+`coupledDot`^†§^ | Time derivative of a coupled variable at q-points
+`coupledDotDot`^†§^ | Second time derivative of a coupled variable at q-points
+`coupledDotDu` | Derivative of the time derivative of a coupled variable at q-points
+`coupledDotDotDu` | Derivative of the second time derivative of a coupled variable at q-points
+`coupledGradientDot`^§^ | Time derivative of the gradient of a coupled variable at q-points
+`coupledGradientDotDot` | Second time derivative of the gradient of a coupled variable at q-points
+
+| Nodal methods for scalar variables | Description |
+| :--- | :--- |
+`coupledNodalValue`^*‡§^ | Value of a coupled variable at nodes
+`coupledNodalDot`^†^ | Time derivative of a coupled variable at nodes
+`coupledNodalDotDot`^†^ | Second time derivative of a coupled variable at nodes
+
+| Methods for vector variables | Description |
+| :--- | :--- |
+`coupledVectorValue`^‡§^ | Value of a coupled vector variable at q-points
+`coupledVectorGradient`^‡§^ | Gradient of a coupled vector variable at q-points
+`coupledCurl`^‡^ | Curl of a coupled vector variable at q-points
+`coupledDiv`^‡^ | Divergence of a coupled vector variable at q-points
+`coupledVectorDot`^†§^ | Time derivative of a coupled vector variable at q-points
+`coupledVectorDotDot`^†^ | Second time derivative of a coupled vector variable at q-points
+`coupledVectorDotDu` | Derivative of the time derivative of a coupled vector variable at q-points
+`coupledVectorDotDotDu` | Derivative of the second time derivative of a coupled vector variable at q-points
+
+| Nodal methods for vector variables | Description |
+| :--- | :--- |
+`coupledNodalValue`^*‡§^ | Value of a coupled vector variable at nodes
+`coupledNodalDot` | Time derivative of a coupled vector variable at nodes
+
+*: These methods are also provided with values from the previous Newton iteration
+by suffixing their name with `PreviousNL`, e.g. `coupledSecondPreviousNL`.
+
+†: These methods are also provided with values from the previous iteration of a
+transient simulation by suffixing their name with `Old`, e.g. `coupledDotOld`.
+
+‡: These methods are also provided with values from the previous or second previous
+iteration of a transient simulation by suffixing their name with `Old` or `Older`,
+respectively, e.g. `coupledDivOld` or `coupledDivOlder`.
+
+§: These methods are also provided with automatic differentiation info by prefixing their
+name with `ad` and preserving `camelCase`, e.g. `adCoupledValue` or `adCoupledVectorDot`.
+
```

Comment:
add a note about "previous" not being available for AD because it doesnt make sense

Reply:
Done?